assets/css/unminified/astra-widget-address.css

```diff
@@ -26,4 +26,8 @@
 }
 .widget-address.widget-address-inline .widget-address-field {
   margin-right:5px;
-}
\ No newline at end of file
+}
+
+.astra-widget-address address, cite {
```

Comment:
This `cite` will change all over the site, need widget specific prefix selector

Reply:
@imnavanath Thanks for pointing it out, Updated the selectors.